src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/StacksTests.cs

```diff
@@ -185,14 +198,20 @@ private static async Task SpeedscopeStacksValidation(AppRunner runner, ApiClient
 
             WebApi.Models.SpeedscopeResult result = await JsonSerializer.DeserializeAsync<WebApi.Models.SpeedscopeResult>(holder.Stream);
 
-            int bottomIndex = result.Shared.Frames.FindIndex(f => f.Name == FormatFrame(ExpectedModule, ExpectedClass, ExpectedFunction));
-            Assert.NotEqual(-1, bottomIndex);
-            string topFrameName = FormatFrame(ExpectedModule, ExpectedClass, ExpectedCallbackFunction);
-            int topIndex = result.Shared.Frames.FindIndex(f => f.Name == topFrameName);
-            Assert.NotEqual(-1, topIndex);
+            string[] framesToFind = [
```

Comment:
nit: Fix the formatting around here e.g. put `[` and `]` on their own lines in the same column as the `s` in `string` and make sure each FormatFrame call it aligned in the same column.

Reply:
We shouldn't need to worry about style nits like this. Our editorconfig should have caused VS to correctly format the document (running format did not fix this). Let's figure out why this regressed.